proposed/clock-meta.md

```diff
@@ -2,21 +2,19 @@
 
 ## 1. Summary
 
-The purpose of using the `ClockInterface` is to provide a standard way to access the system 
-time, that would allow interopability when testing code that relies on the current time 
-rather than relying on installing PHP extensions or use hacks like re-declaring the `time()`
-function in other namespaces. 
+Getting the current time in applications is a rather straightforward thing given the language's native `time()` function (or one of its similar functionalities like `utime()` or `new DateTimeImmutable()` etc.)
+
+Due to the nature of time as constantly progressing it becomes a nuisance to use these functions if you need predictable results, for example when testing.
+
+This `ClockInterface` aims to provide a standard way to consume time, that allows interoperability not only when consuming the "real" time but also when predictable results need to be available. This allows the avoidance of using PHP Extensions for testing or using hacks like re-declaring the `time()` function in other namespaces. 
 
 ## 2. Why Bother?
 
-There are currently a few libraries that provide this functionality, however 
-there is no interopability between these different libraries, as they ship with their own 
-clock interfaces. Symfony provides a package called `symfony/phpunit-bridge` that has a
-`Symfony\Bridge\PhpUnit\ClockMock` class, which allows mocking PHP's built-in time & date 
-functions, however this does not solve mocking calls to `new \DateTimeImmutable()`. It does
-not fully mock time when called from other libraries that rely on the system time. 
-`Cake\Chronos\Chronos` does provide mocking, however it is set via a global (static class 
-property), and this has its own pitfalls as it provides no isolation.
+There are currently a few libraries that provide this functionality, however there is no interopability between these different libraries, as they ship with their own clock interfaces. 
+
+Symfony provides a package called `symfony/phpunit-bridge` that has a `Symfony\Bridge\PhpUnit\ClockMock` class, which allows mocking PHP's built-in time and date functions, however this does not solve mocking calls to `new \DateTimeImmutable()`. It also does not fully mock time when called from other libraries that rely on the system time.
+
+`Cake\Chronos\Chronos` does provide mocking via a static `setTestNow()` method, but this has its own pitfalls as it provides no isolation and must be called again to stop mocking.
```

Comment:
`Chronos` just inherits `setTestNow()` from `Carbon`, the project they forked, which implemented it years before. By respect for @briannesbitt, the original author, please mention `Carbon` instead.

Reply:
This should now be fixed by mentioning Carboin as well as Chronos